src/shapes/OntoMetadataShape.ttl

```diff
@@ -45,9 +45,9 @@ ontom:OntologyMetadataShape a sh:NodeShape ;
 
     sh:property [             
         sh:path [sh:alternativePath (rdfs:label skos:prefLabel dc:title dcterms:title schema:name) ] ; 
-        sh:severity sh:Violation ;
+        sh:severity sh:Info ;
         sh:datatype rdf:langString ;
-        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag."@en ;
+        sh:message "The ontology title is not an rdf:langString, i.e. it does not carry a language tag. We recommend to add one."@en ;
         sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag."@en ;
```

Comment:
```suggestion
        sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag. We recommend to add one."@en ;
```

Reply:
We should then also change 
`sh:description "The ontology title must be an rdf:langString, i.e. it does not carry a language tag."@en`
to 
`sh:description "This shape checks whether the ontology title literal value is an rdf:langString. It will produce an info message if the title does not carry a language tag."@en`

Reply:
I updated the suggestion accordingly.